app/stacks/global/front-door/waf-policy.tf

```diff
@@ -148,6 +148,27 @@ resource "azurerm_frontdoor_firewall_policy" "default" {
       operator       = "Equals"
       selector       = "backOfficeProjectUpdateContent"
     }
+
+    # Exclusions for APPLICS-380 - Exclude all rules for this selector.
+    # POST BO project description and locationDescription, POST document fileName
+    # only applies Back Office fields.

```

Comment:
If these changes are meant for the BO Applics WAF explicitly, I think these exclusions might be currently associated with the wrong WAF atm. The BO Applics WAF is defined on L424. 

Also, are we definitely happy that *all* the checks are switched off for those fields rather than the specific rules that caused the false positives?

Reply:
Thanks, made updates to move into BO section.

We are excluding from all to constantly avoid having to add new exclusions.

Reply:
Sounds good!